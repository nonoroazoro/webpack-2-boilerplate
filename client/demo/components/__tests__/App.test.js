import React from "react";
import renderer from "react-test-renderer";

import App from "../App";

test("App renders correctly.", () =>
{
    const wrapper = renderer.create(<App />).toJSON();
    expect(wrapper).toMatchSnapshot();
});
