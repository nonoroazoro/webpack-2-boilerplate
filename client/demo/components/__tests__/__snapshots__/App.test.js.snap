exports[`test App renders correctly when className is set. 1`] = `
<div
  className="root home">
  <p
    className="title">
    Hello 
    World
    !
  </p>
  <button
    onClick={[Function]}>
    Click: 0
  </button>
</div>
`;

exports[`test App renders correctly. 1`] = `
<div
  className="root">
  <p
    className="title">
    Hello 
    World
    !
  </p>
  <button
    onClick={[Function]}>
    Click: 0
  </button>
</div>
`;
